tests/Basic/BestPracticesTest.php

```diff
@@ -2,26 +2,25 @@
 
 namespace Behat\Mink\Tests\Driver\Basic;
 
+use Behat\Mink\Driver\CoreDriver;
 use Behat\Mink\Tests\Driver\TestCase;
 
 /**
  * This testcase ensures that the driver implementation follows recommended practices for drivers.
  */
 final class BestPracticesTest extends TestCase
 {
-    public function testExtendsCoreDriver()
+    public function testExtendsCoreDriver(): void
     {
         $driver = $this->createDriver();
 
-        $this->assertInstanceOf('Behat\Mink\Driver\CoreDriver', $driver);
-
-        return $driver;
```

Comment:
No idea why this test was returning the driver...so I removed it, hope that's fine?